### ISSUE TITLE
You can click on miasm sprite on the character

### ISSUE THREAD

**Issue by OrangeMann** [mattroks101/InterBay#91]
I don't think that's how it works.
I mean, when your character smells, you're basically have much bigger sprite to click on, don't know if it's crucial.

**Comment by OrangeMann** [comment]
![image](https://user-images.githubusercontent.com/10381296/32062466-45434c3c-ba7d-11e7-9f40-908198557987.png)
![image](https://user-images.githubusercontent.com/10381296/32062478-4cbe688e-ba7d-11e7-9ad0-dc7f0cc59841.png)

**Comment by mattroks101** [comment]
That's how it worked in lifeweb. I don't have a solution in mind yet.

**Comment by OrangeMann** [comment]
Yeah i know, it's part of the character sprite at this point, don't think this can be fixed.

**Comment by Kraseo** [comment]
I personally think they should just give a description on examine and a short-lasting sad moodlet for people around stinky people
